pipeline-module/README.md

```diff
@@ -44,6 +44,15 @@ git add .
 git commit -m "Initialize repo"
 ```
 
+NB: If your version of `git` is too old to support the `git -b` option, you can do the following instead:
```

Comment:
This is the case for folks who are taking the Apple default version of `git` from macOS Catalina.